comms/discovery/rpcz/websocket.go

```diff
@@ -96,3 +99,34 @@ func websocketPush(userId int32, payload []byte) {
 	}
 
 }
+
+func websocketPushAll(rpcJson json.RawMessage, timestamp time.Time) {
+	mu.Lock()
+	defer mu.Unlock()
+
+	for _, s := range websockets {
+		encodedUserId, _ := misc.EncodeHashId(int(s.userId))
+
+		data := struct {
+			RPC      json.RawMessage `json:"rpc"`
```

Comment:
Note this has the side effect of revealing that a blast is not a real DM for sleuthing eyes. I think this is fine given our strategy though.